src/site/_data/event.js

```diff
@@ -23,7 +23,7 @@ const days = [
     title: 'Day 1',
     when: '2020-06-30T16:00Z', // 9am PDT (-7)
     duration: 3 * 60, // minutes
-    videoId: null,
+    videoId: 'oHHSSJDJ4oo',
```

Comment:
remove before submit (unless this is real?) - I see a video unrelated to us in the Netlify preview.